pkg/templates/cel/template.go

```diff
@@ -0,0 +1,108 @@
+package cel
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/cel-go/cel"
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/cel/internal/params"
+)
+
+const (
+	templateKey = "cel-expression"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "CEL",
+		Key:         templateKey,
+		Description: "Flag objects with CEL expression",
+		SupportedObjectKinds: config.ObjectKindsDesc{
+			ObjectKinds: []string{objectkinds.Any},
+		},
+		Parameters:             params.ParamDescs,
+		ParseAndValidateParams: params.ParseAndValidate,
+		Instantiate: params.WrapInstantiateFunc(func(p params.Params) (check.Func, error) {
+			return func(ctx lintcontext.LintContext, object lintcontext.Object) []diagnostic.Diagnostic {
+				msg, err := evaluate(p.Check, object, ctx.Objects())
```

Comment:
It's funny how `object` becomes `subject` in `evaluate()`. Maybe call it `subject` here as well?